ext/opentelemetry-ext-prometheus/src/opentelemetry/ext/prometheus/__init__.py

```diff
@@ -152,33 +152,44 @@ def _translate_to_prometheus(self, metric_record: MetricRecord):
             metric_name = self._prefix + "_"
         metric_name += self._sanitize(metric_record.instrument.name)
 
+        description = getattr(metric_record.instrument, "description", "")
         if isinstance(metric_record.instrument, Counter):
```

Comment:
Curious, is does prometheus not support any other metric type? What about `UpDownCounter`?

Reply:
Prometheus does actually provide a lot of other metric types:

Following the README at: https://github.com/prometheus/client_python#instrumenting the following types are supported:
[x] [Counter](https://github.com/prometheus/client_python#counter)
[ ] [Gauge](https://github.com/prometheus/client_python#gauge)
[x] [Summary](https://github.com/prometheus/client_python#summary) [Adressed in this PR]
[ ] [Histogram](https://github.com/prometheus/client_python#histogram)
[ ] [Info](https://github.com/prometheus/client_python#info)
[ ] [Enum](https://github.com/prometheus/client_python#enum)
[ ] [Labels](https://github.com/prometheus/client_python#labels)

I am not quite sure why the initial implementation only included unknown data sets and counters.
Maybe it was just enough for the use case of the contributor of this package.